app/views/static/index.html.slim

```diff
@@ -7,23 +7,29 @@
       header
         h1 OpenMensa - Die offene Mensa Datenbank.
       .content
+        = link_to about_path do
+          = image_tag image_path('compact.png'), id: 'compact', class: 'illustrate'
         p
           ' OpenMensa ist die kostenfreie Datenbank für Mensen und Cafeterien in Deutschland.
+          ' Wir stellen eine zentralle Austauschplatform zwischen Essensanbietern und Nutzern
+          ' bzw. deren Apps dar.
+          ' Hier findest du die Speisepläne zahlreicher Mensen und Cafés in Deutschland:
+        p.action
+          = link_to about_path, class: "btn btn-action", title: 'Erfahre mehr über OpenMensa' do
+            ' Erfahre mehr über OpenMensa
```

Comment:
I like  the buttons since they do their job and would like to keep them if nothing better pops up even if I'm not sure how well the fit into the rest of the page.